ui/app/mainui/AppMain.qml

```diff
@@ -987,59 +988,68 @@ Item {
                     }
 
                     Repeater {
-                        model: appMain.rootStore.mainModuleInst.sectionsModel
+                        model: SortFilterProxyModel {
+                            sourceModel: appMain.rootStore.mainModuleInst.sectionsModel
+                            filters: ValueFilter {
+                                roleName: "sectionType"
+                                value: Constants.appSection.community
+                            }
+                        }
 
-                        delegate: DelegateChooser {
-                            role: "sectionType"
-                            DelegateChoice {
-                                roleValue: Constants.appSection.community
+                        delegate: Loader {
+                            id: communityLoader
 
-                                delegate: Loader {
-                                    id: communityLoader
-                                    readonly property string sectionId: model.id
+                            readonly property string sectionId: model.id
 
-                                    asynchronous: true
-                                    active: model.loaderActive
```

Comment:
This change conflicts with https://github.com/status-im/status-desktop/pull/9999 ticket
Please, clarify the expectations

Reply:
Please see the commit description

[fix(chat): use loader for ChatMessagesView instead ChatContentView](https://github.com/status-im/status-desktop/pull/10343/commits/7661bafa0e4fa9a30fa29072fffc5933fabe4b2b) 

fixes: https://github.com/status-im/status-desktop/issues/10285

Reply:
The heaviest element in `ChatContentView` is `ChatMessagesView`, so I wrap it in a loader instead. Otherwise the fix you provided [here](https://github.com/status-im/status-desktop/pull/9915/files#diff-716b0e8f898b6b9b6f459e246da466fc36a0a110147aa6cfff50bf4f18c8f049) does not make any sense.

Reply:
It did sense before, but if it does not make sense now, should we remove model.loaderActive code?

Reply:
I don't understand. We still need `loaderActive` to unload `ChatMessagesView` or am I missing something?

Reply:
in `src/app/modules/main/module.nim` -> `onDeactivateSectionAndChatLoader` we have a code for deactivating the section, it is not needed now

Reply:
Oh, right, thanks for clarification. I am unable to locate the requirements of section unloading in [#9235](https://github.com/status-im/status-desktop/issues/9235). Could you please explain if there is a specific rationale behind this decision?

Reply:
@mprakhov I removed the code for deactivating the section: [refactor(chat): remove unused section unloading](https://github.com/status-im/status-desktop/pull/10343/commits/4c2de3786fb8f35fe8392cb81aed75c846397cbe)